src/class-parsely.php

```diff
@@ -727,8 +733,8 @@ private function get_category_name( WP_Post $post_obj, array $parsely_options ):
 		$taxonomy_dropdown_choice = get_the_terms( $post_obj->ID, $parsely_options['custom_taxonomy_section'] );
 		// Get top-level taxonomy name for chosen taxonomy and assign to $parent_name; it will be used
 		// as the category value if 'use_top_level_cats' option is checked.
-		// Assign as "Uncategorized" if no value is checked for the chosen taxonomy.
-		$category = 'Uncategorized';
+		// Assign as the default category name if no value is checked for the chosen taxonomy.
+		$category_name = get_cat_name( get_option( 'default_category' ) );
```

Comment:
Same here. It's unlikely that `get_option('default_category')` fails, but I'd introduce an additional check just in case.